### PR TITLE
KAFKA-2684: Add force option to topic / config command so they can be called programatically

### PR DESCRIPTION
**Pull request by benstopford**
Tiny change to add a force option to the topic and config commands so they can be called programatically without requiring user input.

### CODE REVIEW

File: core/src/main/scala/kafka/admin/ConfigCommand.scala [hunk 1]
```diff
@@ -85,14 +85,15 @@ object ConfigCommand {
     }
   }
 
-  def warnOnMaxMessagesChange(configs: Properties): Unit = {
+  def warnOnMaxMessagesChange(configs: Properties, force :Boolean): Unit = {
```
Comment (ijuma) [inline core/src/main/scala/kafka/admin/ConfigCommand.scala]: Style: space should be after colon, not before.

File: core/src/main/scala/kafka/admin/ConfigCommand.scala [hunk 2]
```diff
@@ -159,6 +160,7 @@ object ConfigCommand {
             .ofType(classOf[String])
             .withValuesSeparatedBy(',')
     val helpOpt = parser.accepts("help", "Print usage information.")
+    val forceOpt = parser.accepts("force", "Suppress console prompts etc")
```
Comment (ijuma) [inline core/src/main/scala/kafka/admin/ConfigCommand.scala]: Do we want the `etc` there?

File: core/src/main/scala/kafka/admin/TopicCommand.scala [hunk 3]
```diff
@@ -308,6 +308,8 @@ object TopicCommand extends Logging {
     val topicsWithOverridesOpt = parser.accepts("topics-with-overrides",
                                                 "if set when describing topics, only show topics that have overridden configs")
 
+    val forceOpt = parser.accepts("force", "Suppress console prompts etc")
```
Comment (ijuma) [inline core/src/main/scala/kafka/admin/TopicCommand.scala]: Same here, do we want `etc` in the message?

**Comment by guozhangwang** [comment]
@benstopford Could you rebase and ping me?

**Comment by ijuma** [comment]
LGTM

**Pull request closed without merging**